django_project/api/api_views/v2/locality_osm_autocomplete.py

```diff
@@ -0,0 +1,23 @@
+# coding=utf-8
+__author__ = 'Anita Hapsari <anita@kartoza.com>'
+__date__ = '15/04/19'
+
+import json
+from django.http import HttpResponse
+from localities_osm.models.locality import LocalityOSMView
+
+
+def locality_autocomplete(request):
```

Comment:
hi @ann26 
please put this into v2/localities
and rename this file into search.py

also please rename function into search_by_name and use APIView